fastify.js

```diff
@@ -917,6 +914,24 @@ function fastify (options) {
     this[kGenReqId] = reqIdGenFactory(this[kOptions].requestIdHeader, func)
     return this
   }
+
+  function acceptHTTPMethod (method, { hasBody = false } = {}) {
```

Comment:
Whenever even a facsimile of named parameters can be used, I prefer to do so. What do you think about?:

```js
function ({ method, acceptBody = true }) {}
```

I also wonder if, _in the future_, we would expand this API to something like:

```js
function ({ method, acceptBody = true, bodyPreProcessor = () => {} }) {}
```

Reply:
It would be better to let `method` on it's own. 
Others is the option to change the behavior of that `method`.

```js
function (method, { acceptBody })
```